spec/features/home_spec.rb

```diff
@@ -66,7 +66,7 @@
         visit '/'
       end
 
-      it 'アクティビティにフォローしている人のチェックイン情報が表示されること', js: true do
+      it 'アクティビティにフォローしている人の記録情報が表示されること', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.